devbuild.sh

```diff
@@ -136,6 +136,11 @@ while :; do
   shift
 done
 
+# Ensure uppercase / lowercase ============================================
```

Comment:
FYI the `^^` and `,,` bashisms don't work on a Mac because Apple apparently doesn't/can't update its version of bash on MacOs (and whatever version they're frozen on doesn't support these features).  That's why in `regional_workflow`, we instead use the `echo_uppercase` and `echo_lowercase` functions, e.g. `COMPILER=$(echo_lowercase $COMPILER)`.  To be able to do that, you have to first source `source_util_funcs.sh` in `regional_workflow/ush`.